### ISSUE TITLE
Would like to offer my help to make Dutch translation file

### ISSUE THREAD

**Issue by PiPo-Sido** [vradarserver/vrs#110]
Hello VRS,

is it possible that i can offer my help to create a Dutch Translation for VRS? If so how can i do it and where do i need to sent the files to?

Thanks Cheers

Danny

**Comment by BallerG** [comment]
VRS already has a Dutch Translation. You just have to go to menu, click on English (United Kingdom) and you can change it to Dutch

**Comment by PiPo-Sido** [comment]
> 

I mean the VRS program it self, not the VRS website.

**Comment by vradarserver** [comment]
@PiPo-Sido Sorry for the delay in replying - that would be nice, thank you. However, I would hold off for a bit until the .NET Core conversion is complete. The plan is to retire the V2 and V3 versions when .NET Core is done so I wouldn't want to spend too much time on those if it can be avoided, and I'm not yet sure what shape the i18n will take for .NET Core. At the moment it's using resx files, but that might change.

So in short - yes please, but hold on a bit :)
